bundles/EcommerceFrameworkBundle/Model/AbstractOrder.php

```diff
@@ -71,23 +72,21 @@ public function setSubTotalPrice($subTotalPrice)
     }
 
     /**
-     * Should return a float
+     * @throws UnsupportedException
      *
-     * @return void
+     * @return float
      */
     public function getSubTotalNetPrice()
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('getSubTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('getSubTotalNetPrice not implemented for ' . get_class($this));
     }
 
     /**
      * @param float $subTotalPrice
      */
     public function setSubTotalNetPrice($subTotalPrice)
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('setSubTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('setSubTotalNetPrice not implemented for ' . get_class($this));
```

Comment:
Again, I'd revert this change for this PR.

Reply:
https://github.com/pimcore/pimcore/pull/7553